core/pom.xml

```diff
@@ -17,7 +17,7 @@
 		<dependency>
 			<groupId>org.apache.httpcomponents</groupId>
 			<artifactId>httpclient</artifactId>
-			<version>4.3.6</version>
+			<version>4.5.13</version>
```

Comment:
Updating this as it was present in an old PR: https://github.com/bitsoex/btcd-cli4j/pull/9